utils/statuses.ts

```diff
@@ -27,6 +27,7 @@ const statuses = [
       "Rihanna walking with joy in the Man Down video",
   },
   { code: 202, title: "Accepted", description: "Rihanna with a bob winking" },
+  {code : 204, title:"No Content", description:"Rihanna sitting and saying why},
```

Comment:
Can you add a `"` mark here so that this syntax is valid?

Reply:
Done fixed it sorry